<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="labObject1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Object 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labObject2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Object 2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboObj1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboObj2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labMsgTxt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Message Text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="strMsgTxt">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Event type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboType">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
